build.gradle

```diff
@@ -128,7 +128,7 @@ jacocoTestReport {
     }
 }
 
-String version = '8.4.2'
+String version = '8.4.3'
```

Comment:
You need to update the test lib version as well right?

Reply:
In build repo. Will do that after this is merged and tag is cut

Reply:
Yeah get confused, let me check.